docs/README.md

```diff
@@ -119,19 +118,58 @@ options:
                         Sets the MFA response to a challenge
   --quiet               Suppress output```
 ```
+Regarding the Okta password, we are fans of automation but do not
+recommend passing in the password to tokendito via plaintext or storing
+it in your environment locally.
 
-## Supported MFA Options
 
--   Native Okta factors (push, call, sms, TOTP) except Biometrics (FIDO
-    webauthn)
+# Environment variables and user configuration
+tokendito supports the use of environment variables and user configuration equivalent to specify the default values for most options.
+
+## Precedence
+Credentials and configuration settings take precedence in the following order:  
+1) Command line options -- Overrides settings in any other location. You can specify \--username, \--role-arn, \--okta-aws-tile, and \--mfa as parameters on the command line.  
+2) Environment variables -- You can store values in your system\'s environment variables. It overrides the configuration file.  
+3) User configuration file -- The user configuration file is updated when you run the command tokendito \--configure. tokendito uses [platformdirs](https://github.com/platformdirs/platformdirs) to store user configuration in the [tokendito.ini](tokendito.ini.md) file. This file can contain the credential details for the default profile and any named profiles.   
+
+## Environment variables and user configuration table
+
+The following table lists the environment variable and user configuration entry equivalent for the given command line option.
+| Command line option | Environment variable | User configuration |
```

Comment:
I think in this table it may be valuable to put in a column with example values?